discretize/utils/interputils.py

```diff
@@ -13,7 +13,7 @@
 except ImportError:
     print("""Interpolation will not work, use setup.py to compile the cython:
 
-        python setup.py build_ext --inplace""")
+        python setup.py install""")
```

Comment:
The only way interputils would fail to import is if it wasn't installed correctly, so technically this would fix that problem.

However if interputils would fail to build, the install command would not happen, thus you wouldn't be able to even import discretize (because it isn't installed), from the python shell outside the source directory.

Essentially this problem shouldn't come up in normal practice as it would throw the error when it is being installed, and not when it is imported.